src/journal/journalctl.c

```diff
@@ -2563,6 +2576,14 @@ int main(int argc, char *argv[]) {
                                         goto finish;
                         }
 
+                        if (arg_until_cursor) {
+                            r = sd_journal_test_cursor(j, arg_until_cursor);
+                            if (r < 0)
+                                    log_error_errno(r, "Failed to test cursor: %m");
+                            else if (r > 0)
+                                    goto finish;
+                        }
```

Comment:
please follow our usual coding style: 8ch indenting. See CODING_STYLE for details